pom.xml

```diff
@@ -151,6 +151,7 @@
 		<dependency>
 			<groupId>org.scijava</groupId>
 			<artifactId>scijava-common</artifactId>
+			<version>2.86.1-SNAPSHOT</version>
```

Comment:
@ktian8 You can remove this change now - we don't want `SNAPSHOT` versions merged back to the main branch.